src/manifest.c

```diff
@@ -633,7 +633,6 @@ struct manifest *load_mom(int version)
 		printf("Failed to load %d MoM manifest\n", version);
 		goto out;
 	}
-	retried = false;
```

Comment:
So, just to be clear, are these the intended failure patterns? These assume the -n option is not passed.
1) if retreive_manifests() fails, return an error
2) if manifest_from_file() fails, retry once; second failure returns an error
3) if download_and_verify_signature() fails, retry once; second failure returns an error

If the above is correct, maybe the failure for step 3 should goto a new label that such that only download_and_verify_signature() is called again? My reasoning is that manifest_from_file() returning success means the MoM was already downloaded and has been parsed correctly.